app/views/mentor_questionnaires/_form.html.erb

```diff
@@ -22,11 +22,20 @@
       <%= form.text_field :company_url %>
     </div>
 
-    <div class="flex flex-col">
-      <%= label_tag 'user[demographic_year_started_ruby]', "Year you started programming in Ruby*" %>
-      <p>Some folk will want to speak to someone who's within touching distance of the beginning of their careers, some will want grizzled veterans.</p>
-      <%= select_tag 'user[demographic_year_started_ruby]', options_for_select((1993..Date.today.year).to_a.reverse, selected: @user.demographic_year_started_ruby), include_blank: true %>
-    </div>
```

Comment:
I don't recall why I did it this way. Nested form seems cleaner so moved demographic_year_started_ruby in there as well